### PR TITLE
added web-graph instruction

### LINKED ISSUE #62: Also looks nice with overlap=false
[here's my config](https://github.com/I60R/dotfiles/blob/cfg/pacwall/.config/pacwall/pacwall.conf)

### PR DESCRIPTION
**Pull request by I60R**
Closes #62

---

Note that the instruction is more complicated than simply adding `overlap=true`

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -330,6 +330,39 @@ Entries of the form
 ``'package-name' [comma-separated-list-of-attributes];``
 in the ``attributes.graph`` setting add attributes to a specific package.
 
+~~~~~~~~~
+Web-graph
+~~~~~~~~~
+
+If you want nice web-graph like on the following image:
+
+.. image:: https://imgur.com/Qc1KiIp.png
```
Comment (Kharacternyk) [inline README.rst:339]: Will it be available forever? Does it make sense to just add the image to the repo, like `screenshot.png`?

File: README.rst [hunk 2]
```diff
@@ -330,6 +330,39 @@ Entries of the form
...
+
+Then create the following config:
+
+.. code-block ::
+
+    hook: "convert ~/.cache/pacwall/pacwall.png -resize 1920x1200! ~/.cache/pacwall/pacwall-fit.png"
```
Comment (Kharacternyk) [inline README.rst:345]: Wallpaper setters usually provide an option to fit the image to the screen. Does your one lack such an option? I think we can let users figure out the resizing themselves, and omit this, as it's not related to the web-graph per se.

File: README.rst [hunk 3]
```diff
@@ -330,6 +330,39 @@ Entries of the form
...
+
+    attributes: {
+        graph: "bgcolor='#16161D' ratio=0.58 overlap=false",
+        package: {
+            common: "shape=point, height=0.02, fontname='Roboto Sans', fontsize=11",
```
Comment (Kharacternyk) [inline README.rst:350]: The `height=0.02` part should be redundant as the size of the point is proportional to the size of the package.

**Review by Kharacternyk — COMMENTED** [review]
Thank you! Could you please take a look at some minor issues I highlighted?

File: README.rst (hunk 2 above)
Comment (I60R) [reply README.rst:345]: I use `oguri` which [doesn't support fit to screen](https://github.com/vilhalmer/oguri#output-options). But perhaps you're right, this has nothing to do with web-graph

File: README.rst (hunk 3 above)
Comment (I60R) [reply README.rst:350]: Okay, I'll remove `height=0.02` as well as font related settings

**Comment by Kharacternyk** [comment]
Thanks!

**Pull request merged**